src/components/RegistrationSuccess/index.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import {
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogContentText,
+  DialogActions,
+  Grow,
+  Typography,
+  makeStyles,
+} from '@material-ui/core';
+import { ReactComponent as FemmecubatorLogo } from './assets/femmecubator-logo.svg';
+import { ReactComponent as CheckCircle } from './assets/check-circle.svg';
+
+const Transition = React.forwardRef(function Transition(props, ref) {
+  return <Grow ref={ref} {...props} />;
+});
+
+const useStyles = makeStyles(() => ({
+  dialogWrapper: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    textAlign: 'center',
+    padding: '2.5rem',
+  },
+  checkCircle: {
+    width: '4rem',
+    height: '4rem',
+    marginTop: '20px',
+  },
+}));
+
+export default function RegistrationSuccess(props) {
+  const { titleText, bodyText, button, openModal } = props;
```

Comment:
deconstruct props in the method signature like below:
`export default function RegistrationSuccess({ titleText = '', bodyText = '', button = {}, openModal = false }) {...`